adapters/medianet/medianet.go

```diff
@@ -89,8 +89,13 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 	}, nil
 }
 
-func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
-	mediaType := openrtb_ext.BidTypeBanner
+func getMediaTypeForImp(bid *openrtb2.Bid, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
+	mediaType, err := getBidMediaTypeFromMtype(bid)
+	if err == nil {
+		return mediaType, nil
+	}
+	mediaType = openrtb_ext.BidTypeBanner
```

Comment:
Should remove this check. If Mtype is not set then please return with err.

Reply:
Removed the entire fallback. Pls check now